examples/preprocessing/plot_dss_array.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+"""Denoising source separation"""
+
+# Authors: Daniel McCloy <drmccloy@uw.edu>
+#
+# License: BSD (3-clause)
+
+from __future__ import division
```

Comment:
we try to avoid this, instead for consistency just do cast `float(x)` when doing division